services/movie.py

```diff
@@ -15,19 +16,24 @@ def get_movies(
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
 
 
+@transaction.atomic
```

Comment:
The `@transaction.atomic` decorator is used correctly to ensure that the `create_movie` function is executed within a database transaction, which helps maintain data integrity.